### ISSUE TITLE
DateFuture may expire

### ISSUE THREAD

**Issue by broad-well** [jrmann100/CountDown#3]
In `background.js`, variable `dateFuture` may expire if Chrome runs overnight. When `GetCount` is called on the second day, `dateFuture` will not represent the correct time, leading to unexpected behaviour.

**Comment by jrmann100** [comment]
Yeah, I figured this out too. Is there any fix other than simply reloading the extension from [chrome://extensions](chrome://extensions)?

**Comment by broad-well** [comment]
My suggestion is to use another `setInterval` on `dateFuture` to constantly check and update the time if necessary.
